eZ/Publish/Core/IO/TolerantIOService.php

```diff
@@ -73,7 +73,7 @@ public function loadBinaryFile($binaryFileId)
     {
         $this->checkBinaryFileId($binaryFileId);
 
-        if ($binaryFileId[0] === '/') {
+        if ($binaryFileId[0] === '/' || (PHP_OS === 'WINNT' && $binaryFileId[1] === ':')) {
```

Comment:
However, I will investigate if this one test itself makes sense if all of this is about http paths :)